styles.css

```
@@ -249,3 +261,61 @@ Shell Path Copy Plugin Styles
+	height: 1em;
+	margin-right: var(--size-4-2);
+	vertical-align: -0.15em;
+	mask-size: contain;
+	mask-repeat: no-repeat;
+	-webkit-mask-size: contain;
+	-webkit-mask-repeat: no-repeat;
+}
+
+.shell-path-copy-formats-enabled .setting-items .setting-item-name::before {
+	--shell-path-copy-state-icon: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24' fill='none' stroke='black' stroke-width='2' stroke-linecap='round' stroke-linejoin='round'%3E%3Ccircle cx='12' cy='12' r='10'/%3E%3Cpath d='m9 12 2 2 4-4'/%3E%3C/svg%3E");
+	background-color: var(--text-accent);
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)